### PR TITLE
feat(upload): upload add arg exclude_path

### PR DESCRIPTION
**Pull request by ma-pony**
upload新增参数exclude_path
用于排除venv, node_modules等不需要的目录

### CODE REVIEW

File: _ext/python/crawlab/actions/upload.py [hunk 1]
```diff
@@ -87,7 +86,8 @@ def upload_dir(dir_path: str, create: bool = True, spider_id: str = None, name=N
         param = cfg.param
     if col_name is None:
         col_name = cfg.col_name
-
+    if exclude_path is None:
+        col_name = cfg.exclude_path
```
Comment (tikazyq) [inline _ext/python/crawlab/actions/upload.py]: why assign `cfg.exclude_path` to `col_name`?

**Review by tikazyq — CHANGES_REQUESTED** [review]
overall seems alright, but please make sure my highlighted comments are addressed, and unit tests have been run through

File: _ext/python/crawlab/actions/upload.py (hunk 1 above)
Comment (ma-pony) [reply _ext/python/crawlab/actions/upload.py]: > why assign `cfg.exclude_path` to `col_name`?

fixed

**Review by tikazyq — APPROVED** [review]

**Comment by ma-pony** [comment]
test failed...

**Comment by tikazyq** [comment]
> test failed...

Perhaps you can checkout why it fails, and run unit tests on your local

**Pull request merged**